bookmarkit.html

```diff
@@ -0,0 +1,241 @@
+    <a class="active" href="#home">Home</a>
+    <a href="#About">About</a>
+    <a href="#FAQ">FAQs</a>
+    <a href="#Discover">Discover Books</a>
+
+  </div>
+
+<!-- Adding a banner with company title and quick description -->
+
+<div class="hero-image">
+  <div class="hero-text">
+    <h1 style="font-size:55px">BookmarkIt <font color = red>. </font></h1>
+    <p>An intuitive online book market. Buy and sell new and used books today.</p>
+
+  </div>
+</div>
+
+<!-- Creating text info sections - Why BookMarkit and How it Works -->
+
+<div class="new-container">
+  <div class="content-pos">
+  <h2> Why BookmarkIt?</h2>
+  <p> Finding books has never been easier. With BookmarkIt, you can find anything from fantasy novels to astronomy textbooks.
+    Our user base is growing faster everyday, generating more options for everyone.
+    BookmarkIt's holistic approach to book buying and selling ensures that people can find the books they need quickly and at a reasonable price.
+    <br>
+    <br>
+    <br>
+
+    <h2> How it Works</h2>
+
+  Once you indicate your intial preferences, we'll list some starting recommendations.
+  Later on, you'll get recommendations based on previous purchases and what you've viewed
+   - all free of cost! </p>
+
+   <p>We'll make sure to take all of your preferences into account. Use keywords, like and dislike buttons, and reviews.
+     You can also follow community members to see what everyone is reading.
+
+  </p>
+  <br>
+  <h2>Questions? Contact Us!</h2>
+
+  <!-- Get form elements, print to console, and then clear the form -->
+
+  <script>
+
+      function getInputValue()
+      {
+          var getName = document.getElementById("name").value;
+          var getEmail = document.getElementById("email").value;
+          var getMessage = document.getElementById("message").value;
+
+          console.log(getName);
+          console.log(getEmail);
+          console.log(getMessage);
+
+          getName = document.getElementById('name').value = '';
+          getEmail = document.getElementById('email').value = '';
+          getMessage = document.getElementById('message').value = '';
+      }
+
+  </script>
+
+<!-- Creating the input boxes of the contact form -->
+
+<input type="text" class="name" id = "name"placeholder="Full Name">
+<input type="text" class="email" id = "email"placeholder="Email">
+<input type="text" class="message" id = "message" placeholder="Message">
+<button class="button" onclick="getInputValue();">Submit</button>
+
+
+
+<!-- Recaptcha v3 - incomplete -->
+
+ <script src="https://www.google.com/recaptcha/api.js"></script>
+
+<!-- Including recaptcha + website button styling was the tricky part (g-recaptcha attributes would clash), but
+here is my understanding of how it works - obtain a site key by registering localhost.
+Create a function that sends information to back end on clicking submit button.
+Then verify that the response human by setting a threshold (on a scale between 0 and 1).
+
```

Comment:
Did you try adding in recaptcha and it clashed or just think it will clash? Can you just modify after recaptcha the button styling?

Reply:
Hi Chloe - I added in the recaptcha following this link: https://developers.google.com/recaptcha/docs/v3 where a sample submit button is created. The issue I was running into had to do with including my button style class with the given code from this page - there would be either a blank rectangle, a very small submit button, or the g-recaptcha button on top of the style attributes I created. Would you like me to try implementing recaptcha by itself?